src/System.Security.Permissions/ref/System.Security.Permissions.cs

```diff
@@ -0,0 +1,1108 @@
+namespace System.Security {
+  public abstract partial class CodeAccessPermission : System.Security.IPermission, System.Security.ISecurityEncodable, System.Security.IStackWalk {
+    protected CodeAccessPermission() { }
+    public void Assert() { }
+    public abstract System.Security.IPermission Copy();
+    public void Demand() { }
+    public void Deny() { }
+    public override bool Equals(object obj) { return default(bool); }
+//    public abstract void FromXml(System.Security.SecurityElement elem);
```

Comment:
@danmosemsft I'm confused by your attribution of nouns :smile:.

`SecurityElement` doesn't currently exist in corefx (this PR is the only place that token exists, per GitHub search). And neither Krzysztof C nor Krzysztof W have done anything with cryptography (at least not since the creation of the System.Security.Cryptography.Algorithms factoring).